compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/SchemaGenerationTest.java

```diff
@@ -203,4 +239,16 @@ private static ProjectEnvironmentBuilder getEnvironmentBuilder() {
         Environment environment = EnvironmentBuilder.getBuilder().setBallerinaHome(DISTRIBUTION_PATH).build();
         return ProjectEnvironmentBuilder.getBuilder(environment);
     }
+
+    private String getStringContentFromGivenFile(Path filePath, String fileName) {
+        try {
+            Stream<String> schemaLines = Files.lines(filePath.resolve(fileName));
+            String schemaContent = schemaLines.collect(Collectors.joining(System.lineSeparator()));
+            schemaLines.close();
+            schemaContent = (schemaContent.trim()).replaceAll("\\s+", "");
```

Comment:
Any particular reason to remove white spaces ?
Don't we have to compare the formatted text ?